fastlane/spec/actions_specs/install_on_device_spec.rb

```diff
@@ -7,6 +7,13 @@
         end").runner.execute(:test)
         expect(result).to eq("ios-deploy  --bundle demo.ipa")
       end
+
+      it "generates a valid command using app name with space" do
+        result = Fastlane::FastFile.new.parse("lane :test do
+          install_on_device(ipa: 'demo app.ipa')
+        end").runner.execute(:test)
+        expect(result).to eq("ios-deploy  --bundle demo\\ app.ipa")
```

Comment:
I tried to solve this "problem" but I saw that it would be an unnecessary disorder. Feel free to settle.